dev/table.h

```diff
@@ -52,46 +52,60 @@ namespace sqlite_orm {
                     using namespace static_magic;
                     using column_type = typename std::remove_reference<decltype(col)>::type;
                     using member_pointer_t = typename column_type::member_pointer_t;
                     using getter_type = typename column_type::getter_type;
                     using setter_type = typename column_type::setter_type;
                     // Make static_if have at least one input as a workaround for GCC bug:
                     // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=64095
-                    if(!res){
-                        static_if<std::is_same<C, member_pointer_t>{}>([&res, &obj, &col](const C& c){
+                    if(!is_found_result) {
+                        static_if<std::is_same<C, member_pointer_t>{}>([&result, &is_found_result, &obj, &col](const C& c){
                             if(compare_any(col.member_pointer, c)){
-                                res = &(obj.*col.member_pointer);
+                                result = (obj.*col.member_pointer);
+								is_found_result = true;
                             }
                         })(c);
                     }
-                    if(!res){
-                        static_if<std::is_same<C, getter_type>{}>([&res, &obj, &col](const C& c){
+                    if(!is_found_result) {
+                        static_if<std::is_same<C, getter_type>{}>([&result, &is_found_result, &obj, &col](const C& c){
                             if(compare_any(col.getter, c)){
-                                res = &((obj).*(col.getter))();
+                                result = ((obj).*(col.getter))();
+								is_found_result = true;
                             }
                         })(c);
                     }
-                    if(!res){
-                        static_if<std::is_same<C, setter_type>{}>([&res, &obj, &col](const C& c){
+                    if(!is_found_result) {
+                        static_if<std::is_same<C, setter_type>{}>([&result, &is_found_result, &obj, &col](const C& c){
                             if(compare_any(col.setter, c)){
-                                res = &((obj).*(col.getter))();
+                                result = ((obj).*(col.getter))();
+								is_found_result = true;
                             }
                         })(c);
                     }
                 });
-                return res;
+                return result;
             }
-            
+
+            /**
+             *  Function used to get field value from object by mapped member pointer/setter/getter
+             */
+            template<class F, class C, typename std::enable_if<!std::is_copy_assignable<F>::value, void*>::type = nullptr>
+            F get_object_field(const object_type &obj, C c) {
+                throw std::system_error(std::make_error_code(orm_error_code::not_implemented));
```

Comment:
So you cannot map a field which has not copy ctor? You break back compatibility with this PR cause one could do it before

Reply:
@fnc12 Any suggestion for resolve this issue?

Reply:
I don't think this code work for not copy assignable before this PR, because for example `*(&(unique_ptr<…>))` is meaningless for storing in DB

Reply:
@fnc12 If you want I can remove this function and back them to past form

Reply:
Look, we have fixed bug in replace. Now you are talking about another bug which has not issue. If there is not issue and no description with sample reproduction code so the bug doesn't exist. I understand that you know the bug but you are the one. I cannot understand you until bug is reported as it must be reported. You reported a bug with replace - it is nice. You offered a PR. The PR did pass a review and I fixed the bug by myself. So you reported a bug - then the bug is fixed. Please report a bug you are talking right now cause I do not understand what are you talking about